kitsune/coolsearch/static/coolsearch/js/components/SearchFormForum.jsx

```diff
@@ -0,0 +1,32 @@
+/*
+React component that controls the forum specific part of the
+Search form.
+Shows the inputs to search the "forum" documents.
+*/
+
+import QueryActionCreator from '../actions/QueryActionCreator.js';
+
+
+export default class SearchFormForum extends React.Component {
```

Comment:
The three `SearchForm`s share a lot in common. I'd make a super class that they all inherit from.